app/models/intake.rb

```diff
@@ -576,15 +575,36 @@ def formatted_mailing_address
     ADDRESS
   end
 
-  def create_intake_document(filename)
-    filename ||= "2020 13614-C with 15080.pdf"
+  def update_or_create_13614c_document(filename)
     pdf_tempfile = pdf
```

Comment:
I know you didn't implement this, but the pattern of having this other method that creates the Tempfile and then it gets turned into something useable/persist-able here seems strange. I wonder how this could be refactored (not now, but something I'd love to hear about an alternate approach to)